README.md

```diff
@@ -22,8 +22,46 @@ Click 'Add Graph', then 'Create a Local Graph'. Enter a graph name, set the pass
 
 That's all you need to do for syncing within the editor to work. To perform interactive queries against the graph, once the graph is Active, click 'Manage' and then on the next page, click 'Open Browser'. If you're unfamiliar with the browser or Neo4j in general, check out the [docs](https://neo4j.com/developer/neo4j-browser/).
 
-Optional Steps:
-To link data to ESCO you will need to install the ESCO TTL file into Neo4J, there are a number of steps that need to be followed and executed, for more information on how to do this view [Neo4J Setup](https://skillsfundingagency.atlassian.net/wiki/spaces/DFC/pages/1491501074/SPIKE+RDF+hosting+-+Neo4j+cluster) on Confluence. (This link is only available to Internal users)
+#### Loading ESCO data
+
+1) Install the APOC plugin - go to plugins -> install
+2) Install the Neosemantics v4 plugin - https://github.com/neo4j-labs/neosemantics - copy jar to plugins directory, add to the end of neo4j.conf - dbms.unmanaged_extension_classes=n10s.endpoint=/rdf
```

Comment:
Don't think this is needed?
We should add the unrestricted and whitelist properties though, see [here](https://github.com/SkillsFundingAgency/dfc-servicetaxonomy-database/blob/master/Resources/Dockerfiles/neo4jescodb.Dockerfile).

Reply:
It was in the existing guide and on the neosemantics docs it says to add it:
![image](https://user-images.githubusercontent.com/61147528/83123878-5b605080-a0cd-11ea-985e-5bdd37c6b41c.png)

Reply:
That's interesting.
We'll have to dig a bit on this setting, as we might have to add it to our neo dockerfile.